### ISSUE TITLE
3.0: Discounts marked as inactive after single use

### ISSUE THREAD

**Issue by mindctrl** [easydigitaldownloads/easy-digital-downloads#6805]
### Expected behavior
A discount code should remain active unless it's configured in a way that deactivates it at some point, such as the max uses setting or the expiration date.

### Actual behavior
They appear to be marked as inactive after a single use, even when configured with no max uses or no expiration date.

![image 2018-07-26 at 1 01 55 pm](https://user-images.githubusercontent.com/1254870/43276854-2505ded4-90d4-11e8-8298-288c3379940c.png)

### Steps to reproduce the behavior
1) Add a new discount code, configured similarly to the screenshot above.
2) Use the discount code
3) Check its status afterwards

I haven't tested many scenarios yet, and I haven't looked at the code, but I've seen this twice now and wanted to log it before I forget.

Here's the row in the table, after being used once.
```
INSERT INTO `wp_edd_adjustments` (`id`, `parent`, `name`, `code`, `status`, `type`, `scope`, `amount_type`, `amount`, `description`, `max_uses`, `use_count`, `once_per_customer`, `min_charge_amount`, `product_condition`, `date_created`, `date_modified`, `start_date`, `end_date`)
VALUES
	(2817, 0, '3ddtest2', '3ddtest2', 'inactive', 'discount', 'global', 'percent', 10.000000000, '', 0, 1, 0, 0.000000000, 'all', '2018-07-26 16:32:50', '2018-07-26 16:33:54', '0000-00-00 00:00:00', '0000-00-00 00:00:00');

```

### Information (if a specific version is affected):
EDD Version (or branch): `release/3.0`

Related, but unsure to what extent:
https://github.com/easydigitaldownloads/easy-digital-downloads/issues/5070
https://github.com/easydigitaldownloads/easy-digital-downloads/issues/5072

**Comment by cklosowski** [comment]
Replicated this on `release/3.0` as well.

**Comment by cklosowski** [comment]
This is my row in the DB after using the discount:
```
10	0	Test Discount One	TEST10	inactive	discount	global	percent	10.000000000	""	0	1	0	0.000000000	all			2018-08-01 20:30:28	2018-08-01 20:41:53	urn:uuid:26aa5bac-717f-458e-9a78-78df98151a73
```

**Comment by sunnyratilal** [comment]
Fixed in f6a68a11cc982b110495b99b62e1aff7ab97901b.
